test/ng/compileSpec.js

```diff
@@ -4219,6 +4227,299 @@ describe('$compile', function() {
     });
 
 
+    describe('one-way binding', function() {
+      it('should update isolate when the identity of origin changes', inject(function() {
```

Comment:
They are necessary. (many other tests have them). I  think it's because the $rootScope etc. "globals" are set up in a module'S return fn.